File: 1/Product.java
public class Product {
	// Atribut
	private String idProduct;	// ID Product
	private int price;			// Harga

	// Konstruktor
	public Product() {
		idProduct = "";
		price = 0;
	}

	/* Set & Get */

	// ID Product
	public void setIDProduct(String idProduct) {
		this.idProduct = idProduct;
	}
	public String getIDProduct() {
		return this.idProduct;
	}

	// Price
	public void setPrice(int price) {
		this.price = price;
	}
	public int getPrice() {
		return this.price;
	}
}

File: 2/C++/Product.cpp
class Product {
	private: 
		// Atribut
		string idProduct;	// ID Product
		int price;			// Harga

	public:
		// Konstruktor
		Product() {
			idProduct = "";
			price = 0;
		}

		/* Set & Get */

		// ID Product
		void setIDProduct(string idProduct) {
			this->idProduct = idProduct;
		}
		string getIDProduct() {
			return idProduct;
		}

		// Price
		void setPrice(int price) {
			this->price = price;
		}
		int getPrice() {
			return price;
		}

		// Destruktor
		~Product(){}
};